_components/form-blocks.md

```diff
@@ -162,8 +162,36 @@ title: Forms Blocks
 <h2>Contact Form</h2>
 
 <div class="preview">
-  <!-- Add HTML markup for example here -->
-  <img src="{{ site.baseurl }}/assets/img/static/Contact-Form_UI_v1.png">
+  
+  <form class="usa-form-large">
+    
+    <fieldset>
+      
+      <legend>Contact us</legend>
+      <p>Email us <a href="javascript:void(0)">info@agency.gov</a>, give us a call at (800)-CALL-GOVT, or send us a message below.</p>
+      
+      <div class="usa-input-grid-medium">
+        <label for="contact-name">Name <span class="usa-additional_text">Optional</span></label>
+        <input id="contact-name" name="contact-name" type="text">
+        
+        <label for="contact-email">Email Address</label>
+        <input id="contact-email" name="contact-email" type="text">
+      </div>
+      
+      <div class="usa-input-grid-large">
+        <label for="contact-comments">Comments</label>
+        <textarea id="contact-comments" name="contact-comments" rows="4" cols="50"></textarea>
+      </div>
+      
+      <div class="usa-input-grid-medium usa-input-buttons_inline">
+        <input type="submit" value="Sign in">
```

Comment:
Thanks! Feel free to include the commit in the comment too, just so we can trace it.

For example, fixed in a2ce15e :)